packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.types.ts

```diff
@@ -64,6 +64,16 @@ export interface ISearchBoxProps extends React.InputHTMLAttributes<HTMLInputElem
    */
   ariaLabel?: string;
 
+  /**
+   * The aria label of the clear button for the SearchBox for the benefit of screen readers.
+   */
+  clearButtonAriaLabel?: string;
+
+  /**
+   * The aria description of the clear button for the SearchBox for the benefit of screen readers.
+   */
+  clearButtonAriaDescription?: string;
```

Comment:
Hi,

we believe that these props be a part of the clearButtonProps property bag. That would make it easier to evolve the APIs. 

clearButtonProps: {
    ariaLabel;
    ariaDescription:
}